### PR TITLE
Replace the unique patients graph with follow up patients for progress tab

### PR DESCRIPTION
**Pull request by vkrmis**
Co-authored-by: Akshay Gupta <kitallis@gmail.com>

### CODE REVIEW

File: app/controllers/api/current/analytics/user_analytics_controller.rb [hunk 1]
```diff
@@ -41,11 +42,14 @@ def total_patients_count
...
-      .group_by_month(:recorded_at, last: MONTHS_TO_REPORT, reverse: true)
-      .count('distinct patient_id')
-      .select { |k, v| k >= first_patient_at_facility.recorded_at.at_beginning_of_month }
+  def follow_up_patients_per_month
+    analytics = FacilityAnalyticsQuery
+                  .new(current_facility, :month, MONTHS_TO_REPORT)
+                  .follow_up_patients_by_period || {}
+
+    dates_for_periods(:month, MONTHS_TO_REPORT).map do |date|
+      [date, analytics_totals(analytics,:follow_up_patients_by_period, date)]
```
Comment (govindkrjoshi) [inline app/controllers/api/current/analytics/user_analytics_controller.rb]:
```suggestion
      [date, analytics_totals(analytics, :follow_up_patients_by_period, date)]
```

File: app/helpers/dashboard_helper.rb [hunk 2]
```diff
@@ -22,8 +22,8 @@ def format_period(period, value)
     period == :month ? value.strftime("%b %Y") : quarter_string(value)
   end
 
-  def analytics_totals(analytics, metric, period)
-    dash_if_zero(analytics.sum { |_, row| row.dig(metric, period) || 0 })
```
Comment (govindkrjoshi) [inline app/helpers/dashboard_helper.rb]: ~Removing dash_if_zero will affect the dashboard views. Is this the required behavior?~

Saw the usage below.

File: app/views/api/current/analytics/user_analytics/show.html.erb [hunk 3]
```diff
@@ -17,18 +17,18 @@
 <% else %>
   <div class="card">
     <%= inline_svg('icon_new_bp.svg', classname: 'icon-float') %>
-    <h3>How many patients have a BP recorded at <%= @current_facility.name %>?</h3>
+    <h3>How many follow-up patients visited <%= @current_facility.name %>?</h3>
```
Comment (govindkrjoshi) [inline app/views/api/current/analytics/user_analytics/show.html.erb]: This string should be internationalized.

Comment (vkrmis) [reply app/views/api/current/analytics/user_analytics/show.html.erb]: Lets pick this up in the transifex PR

**Pull request merged**